### PR TITLE
550 adjust scientific format of generated numbers

### LINKED ISSUE #550: Adjust scientific format of generated numbers
A user expects information about mean energy to be presented as `147.77 MeV`, not at `1.477698e+2`.
On the other hand, some other quantities will be very small (i.e. 1e-10) or very large, so we should have support for scientific format as well.

I'd expect something similar as libdedx offers, where the range units are automatically adjusted.
image

We also need a checkbox where users could select whether units are automatically adjusted or not (i.e. Units: AUTO or FIXED)
By default, units should be AUTO.

### PR DESCRIPTION
**Pull request by ostatni5**
close #550

### CODE REVIEW

**Comment by grzanka** [comment]
@ostatni5 can you add another unit: `Sv`  (sievert) 

image

image

File: src/util/convertUnits/electronvolt.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    to_anchor: 1e+6,
+  },
+  GeV: {
+    name: {
+      singular: 'Giga-Electronvolt',
+      plural: 'Giga-Electronvolts',
+    },
+    to_anchor: 1e+9,
+  },
+};
```
Comment (grzanka) [inline src/util/convertUnits/electronvolt.ts:58]: Add here Tera and Peta as well

File: src/util/convertUnits/baseUnit.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  return newSystem;
+}
+
+export const volumeMeasure: Measure<BaseSystems, BaseUnits> = {
+  systems: {
+    SI: createSISystem('m^3', 3),
+  },
+};
+
+export const fluenceMeasure: Measure<BaseSystems, BaseUnits> = {
```
Comment (grzanka) [inline src/util/convertUnits/baseUnit.ts:100]: rename this to `inverseOfSurfaceMeasure`

**Comment by ostatni5** [comment]
> @ostatni5 can you add another unit: `Sv` (sievert)
> 
> image
> 
> image

Done

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**